test/source/tests/decrypt.ts

```diff
@@ -68,7 +68,7 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithBrowser: Te
     test(
       `decrypt - backup message rendering`,
       testWithBrowser(async (t, browser) => {
-        const threadId = '15f84afa553d8a83';
+        const threadId = '188923a75165a3c8';
```

Comment:
let's not just replace old message with the new one, as some users have backups with old format and backup message rendering should work for this cases too.

we can rename this test to `decrypt - old backup message rendering` and add another one which will check rendering of new backup format

Reply:
Thank you - I'm a bit slow and didn't realize that earlier. I will check both old and new backups.